src/mame/drivers/rockrage.cpp

```diff
@@ -174,11 +176,14 @@ static INPUT_PORTS_START( rockrage )
 	PORT_BIT( 0x04, IP_ACTIVE_HIGH, IPT_SERVICE1 )
 	PORT_BIT( 0x08, IP_ACTIVE_HIGH, IPT_START1 )
 	PORT_BIT( 0x10, IP_ACTIVE_HIGH, IPT_START2 )
-	PORT_DIPNAME( 0x20, 0x20, DEF_STR( Flip_Screen ) )  PORT_DIPLOCATION("SW3:1")
-	PORT_DIPSETTING(    0x20, DEF_STR( Off ) )
+	PORT_DIPNAME( 0x10, 0x10, DEF_STR( Flip_Screen ) )  PORT_DIPLOCATION("SW3:1")
+	PORT_DIPSETTING(    0x10, DEF_STR( Off ) )
 	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
-	PORT_DIPUNUSED_DIPLOC( 0x40, IP_ACTIVE_LOW, "SW3:2" )
-	PORT_SERVICE_DIPLOC( 0x80, IP_ACTIVE_LOW, "SW3:3" )
+	PORT_DIPUNUSED_DIPLOC( 0x20, IP_ACTIVE_LOW, "SW3:2" )
+	PORT_SERVICE_DIPLOC( 0x40, IP_ACTIVE_LOW, "SW3:3" )	
+	PORT_DIPNAME( 0x80, 0x80, "Sound Mode" )            PORT_DIPLOCATION("SW3:4")
+	PORT_DIPSETTING(    0x80, DEF_STR( Stereo ) )
+	PORT_DIPSETTING(    0x00, DEF_STR( Mono ) )
```

Comment:
SW3:4 is read at bit 7 of "P1" below. Move this down there and get rid of the `PORT_BIT( 0x80, IP_ACTIVE_HIGH, IPT_UNKNOWN )` line there. I've confirmed this works through testing.